node/resolvers/utils/checkConfig.ts

```diff
@@ -345,7 +346,10 @@ export const checkConfig = async (ctx: Context) => {
     return null
   }
 
-  if (!settings?.adminSetup?.cartLifeSpan) {
+  if (
```

Comment:
This check should use OR instead of AND

Reply:
This is a check to return to the default settings if no setting is filled in. Therefore, it has to be AND.

Reply:
Yes @Matheus-Aguilar